backend/keepalived/keepalived.go

```diff
@@ -298,54 +304,114 @@ func createRegistrationEvent(entity *types.Entity) *types.Event {
 	return registrationEvent
 }
 
-// handleUpdate sets the entity's last seen time and publishes an OK check event
-// to the message bus.
-func (k *Keepalived) handleUpdate(e *types.Event) error {
-	entity := e.Entity
+func (k *Keepalived) alive(key string, prev liveness.State, leader bool) {
+	lager := logger.WithFields(logrus.Fields{
+		"status":          liveness.Alive.String(),
+		"previous_status": prev.String()})
 
-	ctx := types.SetContextFromResource(context.Background(), entity)
-	if err := k.store.DeleteFailingKeepalive(ctx, e.Entity); err != nil {
-		return err
+	namespace, name, err := parseKey(key)
+	if err != nil {
+		lager.Error(err)
+		return
 	}
 
-	entity.LastSeen = e.Timestamp
+	lager = lager.WithFields(logrus.Fields{"entity": name, "namespace": namespace})
+	lager.Info("entity is alive")
+}
 
-	if err := k.store.UpdateEntity(ctx, entity); err != nil {
-		logger.WithError(err).Error("error updating entity in store")
-		return err
+func (k *Keepalived) dead(key string, prev liveness.State, leader bool) {
+	lager := logger.WithFields(logrus.Fields{
```

Comment:
I propose we change _all_ instances of `logger` to `lager` 🍻

Reply:
But then my strategy for not overriding the logger variable would break :)

Reply:
![giphy](https://user-images.githubusercontent.com/10634078/51057766-6af3fd80-159b-11e9-9ffb-4315c93f72f1.gif)